src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/ElevationSnapshotTest.snap

```diff
@@ -2986,7 +2986,7 @@ org.opentripplanner.routing.algorithm.mapping.ElevationSnapshotTest.directWalk=[
           },
           "generalizedCost": 3648,
           "interlineWithPreviousLeg": false,
-          "legElevation": "0,63.7,1,63.7,1,63.7,11,63.4,21,63.2,31,63.3,41,63.5,51,63.3,61,62.9,71,62.5,81,62.2,91,62.0,101,61.7,111,61.5,121,61.2,131,60.9,141,60.6,151,60.3,160,60.2,240,60.2,1466,60.2,1540,55.5,2299,55.5,2299,0.0,2432,0.0",
+          "legElevation": "0,63.7,1,63.7,1,63.7,11,63.4,21,63.2,31,63.3,41,63.5,51,63.3,61,62.9,71,62.5,81,62.2,91,62.0,101,61.7,111,61.5,121,61.2,131,60.9,141,60.6,151,60.3,160,60.2,240,60.2,1466,60.2,1540,55.5,2299,55.5,2299,NaN,2432,NaN",
```

Comment:
I think there is a problem here 0.0 has been changed to NaN

Reply:
I now realised this was the old behaviour. I don't know how it should really work

Reply:
Yes, I noticed this as well. After looking at [this method](https://github.com/opentripplanner/OpenTripPlanner/blob/17e16dc22573b3807a4d010261f9b5001597e455/src/main/java/org/opentripplanner/routing/algorithm/mapping/GraphPathToItineraryMapper.java#L280) I concluded it was intentional, so I stoped investigation.